src/utils/deploy/hash-files.test.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path')
+
+const test = require('ava')
+
+const { DEFAULT_CONCURRENT_HASH } = require('./constants')
+const { hashFiles } = require('./hash-files')
+
+test('Hashes files in a folder', async (t) => {
+  const dirname = path.resolve(__dirname, '../../../tests/site-with-functions')
```

Comment:
[sand] Can we use the `site-builder` https://github.com/netlify/cli/blob/e9717ff8aa3c52c2a3d7a95f4f86bbb1f5b79668/tests/utils/site-builder.js to keep everything related to the test in the same file?

Reply:
Absolutely, great point. I'll update.

Reply:
Done in https://github.com/netlify/cli/pull/2317/commits/af3925a3a449be20fb1766ecd7a31bb273365441.